### ISSUE TITLE
Support for multiple server URL

### ISSUE THREAD

**Issue by hackzaid** [miguelgrinberg/APIFairy#65]
Hi @miguelgrinberg I see in the core file there's support for multiple Server URLs or if I am mistaken with this line

```
# servers
        servers = [{'url': request.url_root}]
```

How do I set the second URL for instance if I want to set a sandbox and live server URL.

**Comment by miguelgrinberg** [comment]
You can manually edit the OpenAPI data blob in a `process_apispec` handler.

The documentation for the format is [here](https://swagger.io/specification/). To add a server you will need to add your entry in the `servers` key. Something like this:

```python
@apifairy.process_apispec
def my_apispec_processor(spec):
    spec['servers'][0]['description'] = 'Production server'
    spec['servers'].append({'url': 'https://staging.example.com', 'description': 'Staging server'})
    return spec
```

The first line adds a description to the main server (added by APIFairy automatically). The second line adds a second server, also with a description. The definition of what keys can be used to describe a server is [here](https://swagger.io/specification/#server-object).

**Comment by hackzaid** [comment]
Thanks, this worked perfectly.

**Comment by hackzaid** [comment]
Allow me to reopen this.

So, I have set all the neccessary URL for the API.

What is required is that I want to serve the docs on a different URL and maintain the API on the set Server URLs.

According to the documentation, you can set to serve the `/docs` URL with this configuration `APIFAIRY_UI_PATH`?

**Comment by miguelgrinberg** [comment]
I'm sorry, but I don't understand what is the problem that needs addressing. Have you tried changing your docs URL? Is anything not working as documented?

**Comment by hackzaid** [comment]
Exactly, I am looking for a way on how to serve the docs on a different URL, e.g: Host the API on api.somedomain.com and then access the docs on developer.somedomain.com.

**Comment by hackzaid** [comment]
Not to forget, the FileField doesn't support multiple file uploads?

**Comment by miguelgrinberg** [comment]
You are going to serve your documentation yourself. Are you reading the documentation? This is all covered there: https://apifairy.readthedocs.io/en/latest/intro.html#using-a-custom-documentation-endpoint.

> Not to forget, the FileField doesn't support multiple file uploads?

FileField represents one uploaded file.
